### ISSUE TITLE
Fall Guys SIlent Crash

### ISSUE THREAD

**Issue by ItsJustTee** [Heroic-Games-Launcher/HeroicGamesLauncher#3721]
### Describe the bug

Silent crash in the middle of the round.

### Add logs

```text
(11:38:47) INFO:    [Legendary]:        Legendary location: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary
(11:38:47) INFO:    [Gog]:              GOGDL location: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(11:38:47) INFO:    [Connection]:       Connectivity: check-online
(11:38:47) INFO:    [Connection]:       Pinging external endpoints
(11:38:47) INFO:    [Gog]:              Checking for existing gog manifests
(11:38:48) INFO:    [Backend]:          DRM module staus {
  "oimompecagnajdejgnnjijobebaeigek": {
    "status": "new",
    "title": "Widevine Content Decryption Module",
    "version": "4.10.2710.0"
  }
}
(11:38:48) WARNING: [Backend]:          Failed to register protocol with OS.
(11:38:48) INFO:    [Legendary]:        Running command: LEGENDARY_CONFIG_PATH=/home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig/legendary /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(11:38:48) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json --version
(11:38:48) INFO:    [Nile]:             Running command: NILE_CONFIG_PATH=/home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/nile_config /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/nile --version
(11:38:48) INFO:    [Connection]:       Connectivity: online
(11:38:48) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json auth
(11:38:48) ERROR:   [Gog]:              Unable to syncQueued playtime, userData not present
(11:38:48) INFO:    [Backend]:          
System Information:
CPU: 12x 11th Gen Intel(R) Core(TM) i5-11500 @ 2.70GHz
Memory: 33.48 GB (used: 3.06 GB)
GPUs:
  GPU 0:
    Name: NVIDIA Corporation TU117 [GeForce GTX 1650]
    IDs: D=1f82 V=10de SD=400e SV=1458
    Driver: nvidia
OS: Linux Mint 21.3 (Virginia) (linux)

The current system is not a Steam Deck
We are running inside a Flatpak container

Software Versions:
  Heroic: 2.14.1 Flame Emperor Sabo
  Legendary: 0.20.34 Direct Intervention
  gogdl: 1.0.1
  Nile: 1.0.2 Jonathan Joestar

(11:38:48) INFO:    [Frontend]:         Refreshing undefined Library
(11:38:48) WARNING: [Backend]:          refresh not implemented on Sideload Library Manager
(11:38:48) INFO:    [Legendary]:        Refreshing library...
(11:38:48) INFO:    [Legendary]:        Refreshing Epic Games...
(11:38:48) INFO:    [Backend]:          Frontend Ready
(11:38:48) INFO:    [Backend]:          AreWeAntiCheatYet data downloaded
(11:38:48) INFO:    [Backend]:          Checking for current version changelog
(11:38:48) INFO:    [Legendary]:        Running command: LEGENDARY_CONFIG_PATH=/home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig/legendary /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(11:38:50) INFO:    [Winetricks]:       Downloading Winetricks
(11:38:51) INFO:    [Legendary]:        Game list updated, got 2 games & DLCs
(11:38:51) WARNING: [Backend]:          listUpdateableGames not implemented on Sideload Library Manager
(11:38:51) INFO:    [Legendary]:        Checking for game updates: LEGENDARY_CONFIG_PATH=/home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig/legendary /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(11:38:51) INFO:    [Backend]:          Launching Fall Guys (0a2d9f6403244d12969e11da6713137b)
(11:38:51) INFO:    [Backend]:          Preventing display from sleep
(11:38:51) INFO:    [Backend]:          Checking if wine version exists: Wine - Wine-GE-Proton8-26
(11:38:51) INFO:    [Backend]:          Preventing machine to sleep
(11:38:51) INFO:    [Backend]:          Checking if wine version exists: Wine - Wine-GE-Proton8-26
(11:38:51) INFO:    [Backend]:          Checking if wine version exists: Wine - Wine-GE-Proton8-26
(11:38:51) INFO:    [Backend]:          Checking if wine version exists: Wine - Wine-GE-Proton8-26
(11:38:51) DEBUG:   [Backend]:          Running Wine command: wineboot --init
(11:38:52) INFO:    [DXVKInstaller]:    installing dxvk on... /home/tertius/Games/Heroic/Prefixes/default/Fall Guys
(11:38:52) INFO:    [DXVKInstaller]:    dxvk already installed!
(11:38:52) INFO:    [DXVKInstaller]:    installing vkd3d on... /home/tertius/Games/Heroic/Prefixes/default/Fall Guys
(11:38:52) INFO:    [DXVKInstaller]:    vkd3d already installed!
(11:38:52) INFO:    [Legendary]:        Launching Fall Guys: LD_LIBRARY_PATH=/home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/tools/wine/Wine-GE-Proton8-26/lib64:/home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/tools/wine/Wine-GE-Proton8-26/lib:/usr/lib/extensions/vulkan/gamescope/lib HEROIC_APP_NAME=0a2d9f6403244d12969e11da6713137b HEROIC_APP_RUNNER=legendary HEROIC_APP_SOURCE=epic STEAM_COMPAT_INSTALL_PATH=/home/tertius/Games/Heroic/FallGuys LD_PRELOAD= DOTNET_BUNDLE_EXTRACT_BASE_DIR= DOTNET_ROOT= WINEPREFIX="/home/tertius/Games/Heroic/Prefixes/default/Fall Guys" WINEDLLOVERRIDES=winemenubuilder.exe=d WINE_FULLSCREEN_FSR=0 WINEESYNC=1 WINEFSYNC=1 PROTON_EAC_RUNTIME=/home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/tools/runtimes/eac_runtime PROTON_BATTLEYE_RUNTIME=/home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/tools/runtimes/battleye_runtime ORIG_LD_LIBRARY_PATH=/usr/lib/extensions/vulkan/gamescope/lib GST_PLUGIN_SYSTEM_PATH_1_0=/home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/tools/wine/Wine-GE-Proton8-26/lib64/gstreamer-1.0:/home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/tools/wine/Wine-GE-Proton8-26/lib/gstreamer-1.0 WINEDLLPATH=/home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/tools/wine/Wine-GE-Proton8-26/lib64/wine:/home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/tools/wine/Wine-GE-Proton8-26/lib/wine LEGENDARY_CONFIG_PATH=/home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig/legendary /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch 0a2d9f6403244d12969e11da6713137b --wine /home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/tools/wine/Wine-GE-Proton8-26/bin/wine --wrapper /app/bin/gamemoderun --language en
(11:38:52) INFO:    [Backend]:          Stopping Power Saver Blocker
(11:38:53) INFO:    [WineDownloader]:   Updating wine versions info
(11:38:53) INFO:    [WineDownloader]:   Fetching upstream information...
(11:38:53) INFO:    [Backend]:          Starting the Download Queue
(11:38:54) INFO:    [Legendary]:        Found 0 games to update
(11:38:54) INFO:    [WineDownloader]:   wine versions updated
(11:40:12) INFO:    [Legendary]:        Checking if EOS Overlay is enabled: LEGENDARY_CONFIG_PATH=/home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig/legendary /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary eos-overlay info --prefix "/home/tertius/Games/Heroic/Prefixes/default/Fall Guys"
(11:40:12) INFO:    [ExtraGameInfo]:    Using cached ExtraGameInfo data for Fall Guys
(11:40:12) DEBUG:   [Legendary]:        Using cached install info
(11:40:12) INFO:    [ExtraGameInfo]:    Using cached ExtraGameInfo data for Fall Guys
(11:40:12) DEBUG:   [Legendary]:        Using cached install info
(11:40:12) DEBUG:   [Legendary]:        Using cached install info
(11:40:12) INFO:    [Legendary]:        Abort command "LEGENDARY_CONFIG_PATH=/home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig/legendary /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary eos-overlay info --prefix "/home/tertius/Games/Heroic/Prefixes/default/Fall Guys""
(11:41:34) DEBUG:   [Legendary]:        Using cached install info
(11:41:35) INFO:    [Backend]:          Stopping Display Power Saver Blocker
(11:41:36) INFO:    [Frontend]:         Refreshing legendary Library
(11:41:36) INFO:    [Legendary]:        Refreshing library...
(11:41:36) INFO:    [Legendary]:        Refreshing Epic Games...
(11:41:36) INFO:    [Legendary]:        Running command: LEGENDARY_CONFIG_PATH=/home/tertius/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig/legendary /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(11:41:36) DEBUG:   [Legendary]:        Using cached install info
(11:41:36) DEBUG:   [Legendary]:        Using cached install info
(11:41:40) INFO:    [Legendary]:        Game list updated, got 2 games & DLCs
(11:41:41) DEBUG:   [Legendary]:        Using cached install info
(11:41:41) DEBUG:   [Legendary]:        Using cached install info
(11:41:41) DEBUG:   [Legendary]:        Using cached install info
(11:42:13) DEBUG:   [Legendary]:        Using cached install info
(11:42:15) DEBUG:   [Legendary]:        Using cached install info
```

### Steps to reproduce

Quality in the first round and you should get a silent crash.

### Expected behavior

I should be able to play a full game of Fall Guys without silently crashing.

### Screenshots

No screenshots to show this exact issue

### Heroic Version

Latest Stable

### System Information

- OS [e. g. "Ubuntu"]: Linux Mint 21.3

### Additional information

For some dumb reason, if I click the "Report this game not running" button, the log data shows and then the Launcher whitescreens, making it impossible to get the logs from the specific title. Not very convenient. Copying to clipboard does not work and pasting the data in this issue causes my Tab to crash.

**Comment by ItsJustTee** [comment]
My stupid dumbass did not read the areweanticheatyet.com website for the game. I apologise.

**Comment by kode54** [comment]
That site reports it works. It no longer works as of this month's update. They appear to have deleted the Linux runtime from the depot.

**Comment by DINKIN** [comment]
> That site reports it works. It no longer works as of this month's update. They appear to have deleted the Linux runtime from the depot.

can you please explain what ```they appear to have deleted the linux runtime from the depot.```
what that means in laymen terms?

is there a way that we as the end users can resolve this or is this something that upstream developers needs to resolve?

**Comment by lucadoesthings** [comment]
This seems like a problem on your end, since it works fine for me. They deleted the Linux binaries, yes but then it was fine a day later. Do you have e-sync on? That causes quite a few problems.

**Comment by DINKIN** [comment]
> This seems like a problem on your end, since it works fine for me. They deleted the Linux binaries, yes but then it was fine a day later. Do you have e-sync on? That causes quite a few problems.

I was able to get it to run inside BOTTLES without any issue

but I can't seem to get it to work inside heroic games launcher

https://youtu.be/YWFM59cwl4o

**Comment by kode54** [comment]
Please disregard my previous post about the Linux runtime. Apparently the .so they previously included is no longer required to run it. The game was failing due to a network error with the servers that were being sent to the client on startup during the "Downloading Content" phase. They have since pushed an update to this manifest and everything works again.

**Comment by lucadoesthings** [comment]
> > This seems like a problem on your end, since it works fine for me. They deleted the Linux binaries, yes but then it was fine a day later. Do you have e-sync on? That causes quite a few problems.
> 
> I was able to get it to run inside BOTTLES without any issue
> 
> but I can't seem to get it to work inside heroic games launcher
> 
> https://youtu.be/YWFM59cwl4o

The guide on the Heroic wiki works for me, are you doing it a different way?

**Comment by DINKIN** [comment]
> > > This seems like a problem on your end, since it works fine for me. They deleted the Linux binaries, yes but then it was fine a day later. Do you have e-sync on? That causes quite a few problems.
> > 
> > 
> > I was able to get it to run inside BOTTLES without any issue
> > but I can't seem to get it to work inside heroic games launcher
> > https://youtu.be/YWFM59cwl4o
> 
> The guide on the Heroic wiki works for me, are you doing it a different way?

I thought I followed it to the teeer

Maybe we can screen share or video chat through discord or something 

Maybe I'm doing something wrong in one of the steps 

If you have time to help me troubleshoot that would be amazing

**Comment by kode54** [comment]
The runtime may have conflicts with host .NET Runtime if the runtime location environment variables are set globally by your install packages or distribution and creep into Heroic's environment.

**Comment by DINKIN** [comment]
> The runtime may have conflicts with host .NET Runtime if the runtime location environment variables are set globally by your install packages or distribution and creep into Heroic's environment.

can you please dumb that down for me, i don't think i understand what you mean
